exopy_qm/instruments/connections/qm_connection.enaml

```diff
@@ -23,19 +23,15 @@ enamldef QmConnection(BaseConnection):
 
     alias header_dir : header_val.text
 
-    alias serial_number : sr_val.text
```

Comment:
For my own curiosity can you motivate this change ?

Reply:
Despite its name, this variable was holding the path to the configuration file. Since I moved the configuration from the tool browser to a task, I removed it.